src/index.ts

```diff
@@ -3,8 +3,11 @@ import express, { Request, NextFunction, Response } from 'express'
 import handlebars from 'express-handlebars'
 import request from 'request'
 import { authHandler } from './routes/auth'
+import hydraauth from './routes/hydraauth'
+import { getConsent, postConsent } from './routes/consent'
 import errorHandler from './routes/error'
 import dashboard from './routes/dashboard'
+import home from './routes/home'
```

Comment:
What's the difference between home and dashboard?

Reply:
This was a a bit hacky of a route to handle the redirect from kratos (post login) which then accepts the login to hydra and redirects to the authorization page. I was hoping to rethink this before actually opening a PR but looks like k9ert has started it off

Reply:
I've removed the home-route.